tools/slang-test/main.cpp

```diff
@@ -1230,13 +1233,13 @@ TestResult doRenderComparisonTestRun(TestInput& input, char const* langOption, c
 {
     // TODO: delete any existing files at the output path(s) to avoid stale outputs leading to a false pass
 
-    auto filePath999 = input.filePath;
+    auto filePath = input.filePath;
```

Comment:
Just for context/history, sometimes I want to find and work with all the use sites for a variable/field/whatever, so I'll rename it by adding a garbage suffix and then go to all the lines that now have errors.

The great part about that approach is that it is more robust than a typical "find all references" in VS.

The obvious problem is that sometimes you forget to change it back when you are done.